spec/yaml/swiftnav/sbp/solution_meta.yaml

```diff
@@ -0,0 +1,222 @@
+                 values:
+                   - 0: single or averaged ticks
+                   - 1: single or averaged speed
+                   - 2: multi-dimensional ticks
+                   - 3: multi-dimensional speed
+             - 2-3:
+                 desc: Odometer grade
+                 values:
+                   - 0: Low Grade (e.g. quantisized CAN)
+                   - 1: Medium Grade
+                   - 2: Superior Grade
+                   - 3: Reserved
+             - 4-5:
+                 desc: Rate
+                 values:
+                   - 0: Fixed incoming rate
+                   - 1: Incoming when triggered by minimum distance or speed
+                   - 2: Reserved
+                   - 3: Reserved
+             - 6-7:
+                 desc: Reserved
+
+  # the actual type of the elements going into the array solution_inputs in MSG_SOLN_META
+ - SolutionInputType:
+     short_desc: Flags for a given solution input type.
+     desc: |
+       Metadata describing which sensors were involved in the solution.
+       The structure is fixed no matter what the actual sensor type is.
+       The sensor_type field tells you which sensor we are talking about. It also tells you
+       whether the sensor data was actually used or not.
+       The flags field, always a u8, contains the sensor-specific data.
+       The content of flags, for each sensor type, is described in the relevant structures in this section.
+     type: u16
+     fields:
+       - sensor_type:
+           type: u8
+           desc: The type of sensor
+           fields:
+             - 0-2:
+                 desc: The type of sensor
+                 values:
+                   - 0: GNSS Position (see GNSSInputType)
+                   - 1: GNSS Velocity Displacement (see GNSSInputType)
+                   - 2: GNSS Velocity Doppler (see GNSSInputType)
+                   - 3: Odometry Ticks (see OdoInputType)
+                   - 4: Odometry Speed (see OdoInputType)
+                   - 5: IMU Sensor (see IMUInputType)
+                   - 6: Reserved
+                   - 7: Reserved
+             - 3-4:
+                 desc: Whether this sensor input was actually used or not
+                 values:
+                   - 0: unknown
+                   - 1: received and used
+                   - 2: received but not used
+             - 5-7:
+                 desc: Reserved
+       - flags:
+           type: u8
+           units: <Sensor>InputType
+           desc: Refer to each InputType description
+
+ - MSG_SOLN_META:
+     id: 0xFF0F
+     short_desc: Solution Sensors Metadata
+     public: true
+     desc: |
+       This message contains all metadata about the sensors received and/or used in computing the Fuzed Solution.
+       It focuses primarly, but not only, on GNSS metadata.
+     fields:
+       - pdop:
+           type: u16
+           units: 0.01
+           desc: Position Dilution of Precision, as per last received DOPS,
```

Comment:
I would change the phrasing a bit and keep it short:
`Position Dilution of Precision, as per last available DOPS from Starling GNSS engine`